src/app/shared/models/view-mode-icon.ts

```diff
@@ -0,0 +1,4 @@
+export enum ViewModeIcon {
```

Comment:
This file and enum is redundant, please keep things simple and do not create shared classes/files when it is not actually shared across the project. You will always have a time to move to a shared place something when it is needed.